tests/integration/targets/mso_schema_template_bd/tasks/main.yml

```
@@ -405,13 +405,31 @@
       name: VRF
       schema: '{{ mso_schema | default("ansible_test") }}'
       template: Template1
+    intersite_bum_traffic: true
+    arp_flooding: true
+    ep_move_detection_mode: true
   register: nm_add_bd_4
 
+- name: Add bd 4 (normal mode) with no ep_move_detection_mode
```

is this not an update operation?
```suggestion
- name: Update bd 4 with ep_move_detection_mode to False
```